### PR TITLE
Adding DPIF-R3 config

### PR DESCRIPTION
**Pull request by albertkol**
### How to test
1. Be on latest docker-runner and all other repos
2. Git checkout `run-dpif-r3` branch on:
- digital-form-builder-adapter
- funding-service-pre-award-stores
- funding-service-design-frontend
- funding-service-design-notification
3. `make up`
4. Annoying extra step because open date is in the future:
```
docker exec funding-service-pre-award-stores-1 python -m fund_store.scripts.amend_round_dates -q update-round-dates-fab --fund_short_name DPIF --round_short_name R3 -o PAST -d FUTURE
```
5. Go to https://frontend.levellingup.gov.localhost:3008/funding-round/DPIF/R3

### CODE REVIEW

File: fund_store/config/fund_loader_config/FAB/dpif_r3.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        "contact_email": "digitalplanningteam@communities.gov.uk",
+        "contact_phone": "",
+        "contact_textphone": "",
+        "support_times": "9am to 5pm",
+        "support_days": "Monday to Friday (except public holidays)",
+        "instructions_json": {
+            "en": "You must have received an invitation to apply, which should include the fund's prospectus. If you have not received the prospectus, contact digitalplanningteam@communities.gov.uk",
+            "cy": None,
+        },
+        "feedback_link": "https://www.gov.uk/guidance/digital-planning-improvement-fund-privacy-notice",
```
Comment (NarenderRajuB) [inline fund_store/config/fund_loader_config/FAB/dpif_r3.py:103]: @albertkol we may need to make this feedbad link as feedback_link="". There is a fix we will be pushing making feedback link as empty from FAB. as  DPIF doesn't have feedback link, can we do this change manuallu here.

**Review by nuwan-samarasinghe — APPROVED** [review]
LGTM

**Review by NarenderRajuB — CHANGES_REQUESTED** [review]
Please update feedbacklink

File: fund_store/config/fund_loader_config/FAB/dpif_r3.py (hunk 1 above)
Comment (nuwan-samarasinghe) [reply fund_store/config/fund_loader_config/FAB/dpif_r3.py:103]: May be better to verify all the details

**Review by nuwan-samarasinghe — CHANGES_REQUESTED** [review]

**Comment by albertkol** [comment]
I updated the pyproject.toml to bypass the ruff checks on FAB config files. It was throwing errors for line length.
The exception was there already, but it was slightly broken.

File: fund_store/config/fund_loader_config/FAB/dpif_r3.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        "contact_email": "digitalplanningteam@communities.gov.uk",
+        "contact_phone": "",
+        "contact_textphone": "",
+        "support_times": "9am to 5pm",
+        "support_days": "Monday to Friday (except public holidays)",
+        "instructions_json": {
+            "en": "You must have received an invitation to apply, which should include the fund's prospectus. If you have not received the prospectus, contact digitalplanningteam@communities.gov.uk",
+            "cy": None,
+        },
+        "feedback_link": "",
```
Comment (wjrm500) [inline fund_store/config/fund_loader_config/FAB/dpif_r3.py:101]: Please use this feedback link:

https://forms.office.com/e/USFFFFyaNV

As per [this Slack thread](https://communities-govuk.slack.com/archives/C07S3KW31EE/p1733747344621869)

Comment (albertkol) [inline fund_store/config/fund_loader_config/FAB/dpif_r3.py:101]:
```suggestion
        "feedback_link": "https://forms.office.com/e/USFFFFyaNV",
```

Comment (albertkol) [reply fund_store/config/fund_loader_config/FAB/dpif_r3.py]: Sorted! Thanks!

**Review by nuwan-samarasinghe — APPROVED** [review]

**Pull request merged**